lib/services/access_request.go

```diff
@@ -1230,7 +1230,7 @@ func (m *RequestValidator) calculateMaxAccessDuration(req types.AccessRequest) (
 
 	maxDuration := maxDurationTime.Sub(req.GetCreationTime())
 
-	// For dry run requests, the max_duration is set to 7 days.
+	// For dry run requests, the max_duration is set to 14 days.
```

Comment:
```suggestion
	// For dry run requests, use the maximum possible duration.
```

This way the comment won't go out of date every time you change the value.